src/modules/main/components/MainExpertsView.tsx

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import { ExpertBlock } from '../../../lib/components/ExpertBlock';
 import { useStyles } from '../styles/MainExpertsView.styles';
-import cards from '../mockDataExperts';
+import { IExpert } from '../../../lib/types';
+import { store } from '../../../store/store';
+import mockCards from '../mockDataExperts';
+import { LoadData } from '../store/actionTypes';
+
+interface IRootState {
```

Comment:
maybe it would be better to call interface IExpertState

Reply:
no, this is a root state basically :) 
but pls move it to the root reducer, since it's valid for everything and will be extended like:

```
interface IRootState {
main: {
    important: any; // added 
    newest: any;// added 
    experts: IExpert[];
  };
}
```

Reply:
check https://github.com/ita-social-projects/dokazovi-fe/pull/38 - rootReducer

Reply:
done